apps/frontend/src/pages/ComponentsPage.tsx

```diff
@@ -1,16 +1,35 @@
 import React from 'react';
 import Checkbox from 'kepler.gl/dist/components/common/checkbox';
 import styled, { ThemeProvider } from 'styled-components';
-import { Tooltip, Button, TextArea, TextAreaLight, BottomWidgetInner, Input, IconRoundSmall, PanelContent, SidePanelDivider, InputLight, InlineInput, ButtonGroup } from 'kepler.gl/dist/components/common/styled-components';
-import KeplerGlLogo from 'kepler.gl/dist/components/common/logo'
-import Toolbar from 'kepler.gl/dist/components/common/toolbar'
+import {
+  ButtonGroup,
+  Button,
+  SelectionButton,
+  MapControlButton,
+  CheckMark,
+  Tooltip,
+  TextArea,
+  TextAreaLight,
+  TruncatedTitleText,
+  BottomWidgetInner,
+  Input,
+  IconRoundSmall,
+  PanelContent,
+  SidePanelDivider,
+  InputLight,
+  InlineInput,
+  StyledFilterContent,
+  StyledFilteredOption,
+  SelectText,
+  PanelHeaderTitle,
+} from 'kepler.gl/dist/components/common/styled-components';
+import { Trash, Copy, Add } from 'kepler.gl/dist/components/common/icons';
+import KeplerGlLogo from 'kepler.gl/dist/components/common/logo';
```

Comment:
Import might be simplified to search in `kepler.gl/dist/components` or `kepler.gl/dist/components/common` instead.